src/radical/saga/adaptors/irods/irods_replica.py

```diff
@@ -331,41 +340,43 @@ def irods_get_resource_listing(self):
                     # 13 ----
                     entry.name = line[len("resource name: "):].strip()
                     entry.is_resource_group = False
-    
+
                     # TODO: SAVE ALL THE OTHER INFO
                     for i in range(13):
                         cw_result_list.pop(0)
-    
-                    #add our resource to the list
+
+                    # add our resource to the list
                     result.append(entry)
-    
+
                 # check to see if this is an entry for a resource group
                 elif line.startswith("resource group: "):
                     entry.name = line[len("resource group: "):].strip()
                     entry.is_resource_group = True
-    
+
                     # continue processing ilsresc -l results until we
                     # are at the end of the resource group information
                     # ----- is not printed if there are no further entries
                     # so we have to make sure to check we don't pop off
                     # an empty stack too
                     #
-                    # TODO: ACTUALLY SAVE THE LIST OF RESOURCES IN A RESOURCE GROUP
-                    while len(cw_result_list)>0 and (not line.startswith("-----")):
-                        line=cw_result_list.pop(0)
-    
+                    # TODO: SAVE THE LIST OF RESOURCES IN A RESOURCE GROUP
+                    while len(cw_result_list) > 0 and \
```

Comment:
```suggestion
                    while cw_result_list and \
```
I think pep8 does not like checking if lists have elements by a len check.